### ISSUE TITLE
Explore the effect of batchnorm

### ISSUE THREAD

**Issue by bw4sz** [weecology/DeepForest#477]
The batchnorm in the default retinanet model is a headache and a source of worry. We currently turn on batchnorm update statistics in validation step in main.py, but obviously don't allow the weights to update.

```
        #Get loss from "train" mode for batch norm statistics, but don't allow optimization
        self.model.train()
        with torch.no_grad():
            loss_dict = self.model.forward(images, targets)
```

This is awkward and looks questionable. Even for just appearances sake I'd like so hard evidence that batchnorm is a needed part of this. Relates to the whole 'batchnorm is evil magic' theme.

https://patrick-llgc.github.io/Learning-Deep-Learning/assets/papers/Devils_in_BatchNorm_yuxin_wu.pdf
